src/test/java/com/algolia/search/integration/client/TimeoutTest.java

```diff
@@ -16,10 +16,10 @@
   void testUnreachableHostExceptions() {
     try (SearchClient client =
         new SearchClient(
-            (SearchConfig)
-                new SearchConfig(ALGOLIA_APPLICATION_ID_1, ALGOLIA_API_KEY_1)
-                    .setConnectTimeOut(2)
-                    .setReadTimeOut(2))) {
+            new SearchConfig.Builder(ALGOLIA_APPLICATION_ID_1, ALGOLIA_API_KEY_1)
```

Comment:
Ah! Way better than the forced-downcasting previous version :)

Reply:
Sure :D